src/OrchardCore/OrchardCore.Media.Core/Events/DefaultMediaFileStoreCacheEventHandler.cs

```diff
@@ -12,23 +12,11 @@ public DefaultMediaFileStoreCacheEventHandler(IMediaFileStoreCacheFileProvider m
     }
 
     public override Task MediaDeletedDirectoryAsync(MediaDeletedContext context)
-    {
-        _mediaFileStoreCacheFileProvider.TryDeleteDirectoryAsync(context.Path);
-
-        return Task.CompletedTask;
-    }
+        => _mediaFileStoreCacheFileProvider.TryDeleteDirectoryAsync(context.Path);
```

Comment:
I think we accepted the rule to use block body for methods, ctor and conversions. Only use expression body for properties and indexers.

Reply:
I don't think there was an objection into using expression for one line like this.

Reply:
I looked at the editor config and watched the meeting. You are correct, for methods we "can" use expression bodies when it is on a single line.